openshift_day2_configurator/utils/resources.py

```diff
@@ -10,7 +10,7 @@ def create_ocp_resource(ocp_resource: Any, logger: logging.Logger) -> Dict[str,
         if ocp_resource.exists:
             return {"res": False, "err": f"{resource_kind} {name} already exists"}
         else:
-            logger.debug(f"Create LDAP groups sync cluster role {name}")
+            logger.debug(f"Create {resource_kind} {name}")
```

Comment:
I think the log can be removed since the wrapper already logging this.

Reply:
done